Lib/test/test_hashlib.py

```diff
@@ -403,6 +415,7 @@ def test_blocksize_name_blake2(self):
         self.check_blocksize_name('blake2b', 128, 64)
         self.check_blocksize_name('blake2s', 64, 32)
 
+
```

Comment:
nitpick: PEP 8 suggests only 1 empty line between functions, you may revert this unrelated change.